### PR TITLE
[fix](bloomFilter)fix data type supported behavior same in FE and BE

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes
fix create table stmt for properties: bloom_filter_columns,  which should same with BE behavior 
Issue Number: close #xxx

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java [hunk 1]
```diff
@@ -642,11 +642,18 @@ public static Set<String> analyzeBloomFilterColumns(Map<String, String> properti
...
+                        // string/varchar/char/variant
+                        // date/datetime/datev2/datetimev2
+                        // decimal/decimal32/decimal64/decimal128I/decimal256
+                        // ipv4/ipv6
+                        //
                         // key columns and none/replace aggregate non-key columns support
-                        if (type == PrimitiveType.TINYINT || type == PrimitiveType.FLOAT
-                                || type == PrimitiveType.DOUBLE || type == PrimitiveType.BOOLEAN
-                                || type.isComplexType()) {
+                        if (!((type ==  PrimitiveType.SMALLINT || type == PrimitiveType.INT
```
Comment (eldenmoon) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java:653]: maybe add a function isSupportedBloomFilter(PrimitiveType type)

**Comment by amorynan** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java (hunk 1 above)
Comment (amorynan) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java]: done!

**Review by eldenmoon — APPROVED** [review]
LGTM

**Review by starocean999 — APPROVED** [review]

**Pull request merged**